package datto_linux_client;

import "start_backup_request.proto";
import "stop_backup_request.proto";
import "progress_backup_request.proto";

message Request {
    enum RequestType {
        START_BACKUP = 1;
        STOP_BACKUP = 2;
        PROGRESS = 3;
    }

    required RequestType type = 1;

    optional StartBackupRequest start_backup_request = 2;
    optional StopBackupRequest stop_backup_request = 3;
    optional ProgressBackupRequest progress_request = 4;
}
